AppActionVisualizer/app/src/main/java/com/example/appactionvisualizer/constants/Constant.java

```diff
@@ -10,54 +9,84 @@
-  public final static String ENTITY_URL = "url";
+  public static final String ENTITY_FIELD_IDENTIFIER = "identifier";
+  public static final String ENTITY_FIELD_NAME = "name";
+  public static final String ENTITY_ITEM_LIST = "itemListElement";
+  public static final String ENTITY_URL = "url";
 
-  public final static int MAX_RESULTS = 5;
+  public static final int MAX_RESULTS = 5;
 
   // Activity result code
-  public final static int SELECT_ADDRESS = 0x1;
-  public final static int INPUT_PARAMETER = 0x10;
+  public static final int SELECT_ADDRESS = 0x1;
+  public static final int INPUT_PARAMETER = 0x10;
 
   // Some Actions type Constant String values
-  public final static Set<String> COMMON_SET = new HashSet<>(Arrays.asList("actions.intent.OPEN_APP_FEATURE", "actions.intent.GET_ACCOUNT", "actions.intent.GET_ORDER", "actions.intent.GET_THING"));
-  public final static Set<String> FINANCE_SET = new HashSet<>(Arrays.asList("actions.intent.CREATE_MONEY_TRANSFER", "actions.intent.CREATE_TRADE_ORDER", "actions.intent.GET_FINANCIAL_POSITION", "actions.intent.GET_STOCK_QUOTE", "actions.intent.GET_INVOICE", "actions.intent.PAY_INVOICE"));
-  public final static Set<String> FOOD_SET = new HashSet<>(Arrays.asList("actions.intent.ORDER_MENU_ITEM"));
-  public final static Set<String> TRANSPORTATION_SET = new HashSet<>(Arrays.asList("actions.intent.CANCEL_TAXI_RESERVATION", "actions.intent.CREATE_TAXI_RESERVATION", "actions.intent.GET_TAXI_RESERVATION"));
-
+  public static final Set<String> COMMON_SET =
+      new HashSet<>(
+          Arrays.asList(
+              "actions.intent.OPEN_APP_FEATURE",
+              "actions.intent.GET_ACCOUNT",
+              "actions.intent.GET_ORDER",
+              "actions.intent.GET_THING"));
+  public static final Set<String> FINANCE_SET =
+      new HashSet<>(
+          Arrays.asList(
+              "actions.intent.CREATE_MONEY_TRANSFER",
+              "actions.intent.CREATE_TRADE_ORDER",
+              "actions.intent.GET_FINANCIAL_POSITION",
+              "actions.intent.GET_STOCK_QUOTE",
+              "actions.intent.GET_INVOICE",
+              "actions.intent.PAY_INVOICE"));
+  public static final Set<String> FOOD_SET =
+      new HashSet<>(Arrays.asList("actions.intent.ORDER_MENU_ITEM"));
+  public static final Set<String> TRANSPORTATION_SET =
+      new HashSet<>(
+          Arrays.asList(
+              "actions.intent.CANCEL_TAXI_RESERVATION",
+              "actions.intent.CREATE_TAXI_RESERVATION",
+              "actions.intent.GET_TAXI_RESERVATION"));
 
   @StringRes
-  public static final int[] TAB_TITLES = new int[]{R.string.tab_text_statistics, R.string.tab_text_all, R.string.tab_text_1, R.string.tab_text_2, R.string.tab_text_3, R.string.tab_text_4, R.string.tab_text_5};
+  public static final int[] TAB_TITLES =
+      new int[] {
+        R.string.tab_text_statistics,
+        R.string.tab_text_all,
+        R.string.tab_text_1,
```

Comment:
would you want to rename this to something like tab_text_common maybe? and same for the following

Reply:
Sure, fixed.